docs/schema.md

```diff
@@ -1,37 +1,38 @@
-## Database schema in Scalar DB
+# Database schema in Scalar DB
 
 Scalar DB has its own data model and schema, that maps to the implementation specific data model and schema.
 Also, it stores internal metadata for managing transaction logs and statuses.
-This document briefly describes how the data model and schema are mapped between Scalar DB and other implementations.
+This document briefly explains the Scalar DB data model, how data types are mapped between Scalar DB and the other database implementations, internal metadata, and how Scalar DB database schema can be defined.
 
-## Scalar DB and Cassandra
+## Data model
 
-The data model in Scalar DB is quite similar to the data model in Cassandra, except that in Scalar DB it is more like a simple key-value data model and it does not support secondary indexes.
-The primary key in Scalar DB is composed of one or more partition keys and zero or more clustering keys. Similarly, the primary key in Cassandra is composed of one or more partition keys and zero or more clustering columns.
+The data model of Scalar DB is a multi-dimensional map based on the key-value data model. A logical record is composed of partition-key, clustering-key and a set of values. The value is uniquely mapped by a primary key composed of partition-key, clustering-key and value-name as described in the following scheme.
 
+(partition-key, clustering-key, value-name) -> value-content
 
-Data types supported in Scalar DB are a little more restricted than in Cassandra.
-Here are the supported data types and their mapping to Cassandra data types.
+For each database implementation, there is an adapter that converts the database specific data model into the Scalar DB data model, thus; users usually don't need to care about how it is converted and can design database schema on the basis of the Scalar DB data model. 
 
-|Scalar DB  |Cassandra  |
-|---|---|
-|BOOLEAN  |boolean  |
-|INT  |int  |
-|BIGINT  |bigint  |
-|FLOAT  |float  |
-|DOUBLE  |double  |
-|TEXT  |text  |
-|BLOB  |blob  |
+## Data type mapping between Scalar DB and the other databases
 
-## Internal metadata in Scalar DB
+Here are the supported data types in Scalar DB and their mapping to the data types of other databases.
 
-Scalar DB executes transactions in a client-coordinated manner by storing and retrieving metadata stored along with the actual records.
+| Scalar DB | Cassandra | Cosmos DB | DynamoDB              | MySQL    | PostgreSQL       | Oracle        | SQL Server      |
+| --------- | --------- | --------- | --------------------- | -------- | ---------------- | ------------- | --------------- |
+| BOOLEAN   | boolean   | boolean   | boolean               | boolean  | boolean          | number(1)     | bit             |
```

Comment:
For Cosmos DB, use the following mapping.
Since Cosmos DB is schema-less and a record is expressed in a JSON format, data types are mapped to JSON types.

BOOLEAN -> boolean (JSON)
INT -> number (JSON)
BIGINT -> number (JSON)
FLOAT -> number (JSON)
DOUBLE -> number (JSON)
TEXT -> string (JSON)
BLOB -> string (JSON)

Reply:
Thank you. Updated in 9cc15ea